config/flag.to.vcf.convert.ini

```diff
@@ -139,4 +139,14 @@ info=1
 val=1
 type=Float
 id=ASMD
-description=A soft flag median alignement score of reads showing the variant allele
\ No newline at end of file
+description=A soft flag median alignement score of reads showing the variant allele
+
+[cavemanMatchNormalProportionFlag]
+info=0
+id=CMNP
+description=Tumour sample mutant allele proportion - normal sample mutant allele proportion < maxCavemanMatchedNormalProportion (differs from MNP in using CaVEMan only seen reads as per VCF)
+
+[withinGapRangeFlag]
+info=0
+id=GAP
+description=If variant is within withinXBpOfDeletion of a deletion/gap seen in reads without the variant, the deletion is present in at least minGapPresentInReads percent of total reads and no variant showing reads show the deletion. 
```

Comment:
> If variant is within withinXBpOfDeletion of a deletion/gap seen in reads without the variant, the deletion is present in at least minGapPresentInReads percent of total reads and no variant showing reads show the deletion.

If variant is within withinXBpOfDeletion of an indel in reads without the variant and the indel is present in at least minGapPresentInReads percent of total reads and no variant reads have the indel.

Reply:
Corrected in 9d4611f